aws_grant_user_access/src/process_event.py

```diff
@@ -22,10 +22,13 @@
 config = Config()
 
 
-def process_event(event: Dict[str, Any], context: Any) -> None:
+def process_event(event: Dict[str, Any], context: Any) -> str:
```

Comment:
`process_event()` is only used in `main.py` and the return value is not used there. I'd expect the return type to be `None`